db/migrate/20180501155601_update_land_registry_titles.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateLandRegistryTitles < ActiveRecord::Migration[5.2]
+  def change
+    list = SubscriberList.where("title like '%Land Registry%' and title not like '%HM Land Registry%'")
```

Comment:
I like to make the SQL part (i.e. `like` and `not like`) in capitals, but that's just personal preference really.